### ISSUE TITLE
How to configure rules to exclude/ignore

### ISSUE THREAD

**Issue by smlombardi** [SublimeLinter/SublimeLinter-csslint#10]
I have Googled around and can't find any sample files or docs on how to correctly set SublimeLinter prefs file as to which csslint rules should be ignored.  I currently have this:

```
"csslint": {
                "@disable": false,
                "args": [],
                "errors": "",
                "excludes": [
                    "ids",
                    "box-model",
                    "font-sizes",
                    "unique-headings",
                    "import",
                    "qualified-headings",
                    "adjoining-classes",
                    "overqualified-elements"
                ],
                "ignore": "",
                "ignored": [],
                "warnings": ""
            },
```

I have tried exclude and ignored, but the errors are still flagged.

**Comment by Fanfan92** [comment]
I was searching for that too. The doc is saying that the ignore setting should be "A comma-separated list of rules to ignore". I thought it was waiting for a JSON array [ ], but finally understood that the list should be a comma-separated string, without spaces :
```
"csslint": {
                "@disable": false,
                "ignore": "important,ids"
            }
```
Here I ignored 2 rules : the usage of !important and the usage of ids. You need to find the exact name of the rules by looking at each rule here https://github.com/CSSLint/csslint/wiki/Rules

Also, another way of doing that is using the "ignore_match" option, available for all linters. It expects an array of Regex, each regex representing the error message itself : 
```
"csslint": {
                "@disable": false,
                "ignore_match": [
                    ".+important",
                    ".*use ID.*",
                    "Rule is empty"
                ]
            }
``` 
I like this second method and use it for other linters too, as you only have to copy the error message (which is written as output on the bottom line in Sublime Text) instead of having to search for the exact rule name.
